core/types/src/attributes.rs

```diff
@@ -145,7 +145,7 @@ bitflags! {
         /// PKRU(Protection Keys Register Userspace) state
         const PKRU = SGX_XFRM_PKRU as u64;
         /// AMX(Advanced Matrix Extensions), including XTILEDATA(0x40000) and XTILECFG(0x20000)
-        const AMX = SGX_XFRM_LEGACY as u64;
+        const AMX = SGX_XFRM_AMX as u64;
         /// Reserved for future flags.
         const RESERVED = (!(Self::LEGACY.bits() | Self::AVX.bits() | Self::AVX_512.bits() | Self::PKRU.bits() | Self::AMX.bits()));
```

Comment:
Double checking these values, `MPX` is omitted from the RESERVED value per https://github.com/intel/linux-sgx/blob/1bf092a3894135faea1f6e4f61df0fe96dcb736b/common/inc/sgx_attributes.h#L55